pkgs/applications/office/hledger-check-fancyassertions/default.nix

```
@@ -0,0 +1,40 @@
+{lib, stdenv, haskellPackages, fetchurl, writers, ghc}:
+
+stdenv.mkDerivation rec {
+  pname = "hledger-check-fancyassertions";
+  version = "1.23";
+
+  src = fetchurl {
+    url = "https://raw.githubusercontent.com/simonmichael/hledger/hledger-lib-${version}/bin/hledger-check-fancyassertions.hs";
+    sha256 = "08p2din1j7l4c29ipn68k8vvs3ys004iy8a3zf318lzby4h04h0n";
+  };
+
+  dontUnpack = true;
```

Another phase that easily can be saved.

```suggestion
  dontUnpack = true;
  dontBuild = true;
```